avr-hal-generic/src/lib.rs

```diff
@@ -1,5 +1,5 @@
 #![no_std]
-#![feature(asm)]
+#![feature(asm, llvm_asm)]
```

Comment:
We can drop the `asm` feature for now as everything is using the old
syntax at this time.